configs/model/im2im/segmentation_plugin.yaml

```diff
@@ -27,10 +27,8 @@ task_heads:
         _target_: cyto_dl.models.im2im.utils.postprocessing.ActThreshLabel
         rescale_dtype: numpy.uint8
       prediction:
-        _target_: cyto_dl.models.im2im.utils.postprocessing.ActThreshLabel
-        activation:
-          _target_: torch.nn.Sigmoid
-        rescale_dtype: numpy.uint8
+        _target_: cyto_dl.models.im2im.utils.postprocessing.AutoThreshold
+        method: "otsu"
```

Comment:
Two questions:
1. Is it possible to add support in `AutoThreshold` for a `None` value for `method`? That way if the user doesn't want to use any postprocessing we can set it in the same way we would normally instead of deleting this step from the config file.
2. AFAIK, `otsu` is not an attribute of `skimage.filters` (`threshold_otsu` is, though). Does the threshold_ prefix get added somewhere in the processing for this?